CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-web-session ChangeLog
 
+### 1.6.1 - TBD
+
+### Changed
+- Replaced `axios` with `@digitalbazaar/http-client` as we are sunsetting axios.
+- Changed `SessionService.js` to use http-client instead of `axios`.
+
+### Removed
+- Removed `axios` from the library.
+
+### Added
+- Added `@digitalbazaar/http-client` to the library.
+
```

Comment:
The changed section alone will suffice here.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-web-session/pull/12/commits/7add4e43549db8089cf69a26daba4220f1e78d9a